src/Action.hs

```diff
@@ -135,7 +136,7 @@ import LifeCycle
 import Logger
 import Logger.EventLog
 import Persistent
-import Persistent.Api
+import Persistent.Api as Persist
```

Comment:
would have been nice to have a separate commit for this and all the implications.